src/app/search/FilterHelp.tsx

```diff
@@ -72,11 +94,23 @@ export default function FilterHelp() {
   );
 }
 
-function FilterExplanation({ filter }: { filter: FilterDefinition }) {
+function FilterExplanation({
+  filter,
+  suggestionContext,
+}: {
+  filter:
+    | FilterDefinition<Loadout, LoadoutFilterContext, LoadoutSuggestionsContext>
+    | FilterDefinition;
+  suggestionContext: LoadoutSuggestionsContext | SuggestionsContext;
+}) {
   const dispatch = useDispatch();
-  const suggestions = Array.from(
-    new Set([...generateGroupedSuggestionsForFilter(filter, true, {})]),
+  let suggestions = Array.from(
+    new Set([...generateGroupedSuggestionsForFilter(filter, suggestionContext, true)]),
```

Comment:
This seems like it would be useful for all invocations of `generateGroupedSuggestionsForFilter`, is it worth folding it in? I did a quick search and it seems to be used in a few places so you can probably answer best as to whether we want that.